.github/workflows/AssignReviewers.yml

```diff
@@ -0,0 +1,31 @@
+## @file
+# Assign reviewers from a REVIEWERS file using CODEOWNERS syntax
+#
+# Copyright (c) 2022, Intel Corporation. All rights reserved.<BR>
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+# Copied from https://github.com/mdkinney/github-action-assign-reviewers
+# commit 77731e1
+
+name: Assign reviewers from a REVIEWERS file using CODEOWNERS syntax
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, reopened, ready_for_review]
+    branches:
+      - main

```

Comment:
I may have missed it. Is there a plan to rename `master` to `main` in edk2-platforms?

Reply:
Oops. No, that's a mistake. I'll fix it.